MPC/mpc_contribute.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "What is... your name?"
+read NAME
+
+echo "NO ONE EXPECTS THE... Entropy source I'm asking you:"
+read RANDOM
+
+echo "Thanks, hang on a bit!"
+
+if ! docker --version; then
+    curl -fsSL https://get.docker.com -o get-docker.sh
+    sh get-docker.sh
+fi
+
+docker build -t contribute --build-arg NAME=${NAME} --build-arg RANDOM=${RANDOM} .
```

Comment:
Add "${NAME}" and "${RANDOM}" instead. Otherwise, it will fail if one enters a space as one of the characters